File: 1293-three-consecutive-odds/three-consecutive-odds.js
/**
 * @param {number[]} arr
 * @return {boolean}
 */
var threeConsecutiveOdds = function(arr) {
    var i = 0;
        while(i < arr.length){
            if(arr[i]%2==1){
                var c = 0;
                while(i < arr.length && arr[i]%2==1){
                    i++;
                    c++;
                }
                if(c >= 3) {return true;}
                
            }
            else i++;
        }
        return false;
};

File: 1293-three-consecutive-odds/three-consecutive-odds.cpp
class Solution {
public:
    bool threeConsecutiveOdds(vector<int>& arr) {
        int i = 0;
        while(i < arr.size()){
            if(arr[i]%2==1){
                int c = 0;
                while(i < arr.size() and arr[i]%2==1){
                    i++;
                    c++;
                }
                if(c >= 3) return true;
                
            }
            else i++;
        }
        return false;
    }
};